src/lib/tests/e2e/dashboard.e2e.test.js

```diff
@@ -46,3 +48,67 @@ describe('dashboard e2e', () => {
     server.close();
   });
 });
+
+
+describe('port, config, appId, serverURL, masterKey, appName, graphQLServerURL options should not be ignored.', () => {
+  it('Should start with port 4041.', async () => {
+    const result = await startParseDashboardAndGetOutput(['--port', '4041']);
+
+    expect(result).toContain('The dashboard is now available at http://0.0.0.0:4041/');
+  });
+
+  it('Should return an error message if config and appId options are provided together.', async () => {
```

Comment:
Could you combine all the new tests that should yield the same error message into a single test?

```js
it('should reject to start if config and other options are combined', async () => {
...

const args = [
  '--appId',
  '--serverURL',
  ...
];

for (const arg of args) {
  const result = await startParseDashboardAndGetOutput(['--config', 'helloworld', arg, 'helloworld']);
  expect(...)
}
```

Reply:
Done. It looks nice now.